src/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { Loader } from './components/Loader';
+import { useEffect, useState } from 'react';
+import { Person } from './types';
+import { getPeople } from './api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [error, setError] = useState('');
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => {
+        setIsLoading(true);
+        setPeople(data);
+      })
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);

  useEffect(() => {
     setIsLoading(true);
     
     getPeople()
      .then(setPeople)
```
You should set loading when you start loading something, and then remove in finally when finish